launcher/ui/dialogs/ModUpdateDialog.cpp

```diff
@@ -236,6 +236,11 @@ auto ModUpdateDialog::ensureMetadata() -> bool
         if (skip_rest)
             continue;
 
+        if (candidate->type() == ResourceType::FOLDER) {
+            m_failed_metadata.append({ candidate, tr("This is a folder.") });
```

Comment:
```suggestion
```
if you have a single folder this will show every time you check for updates with nothing selected